app/services/nomis/elite2/offender_api.rb

```diff
@@ -98,11 +98,15 @@ def self.get_bulk_sentence_details(booking_ids)
       end
 
       def self.get_image(booking_id)
+        # This method returns the raw bytes of an image, the equivalent of loading the
+        # image from file on disk.
         details_route = '/elite2api/api/offender-sentences/bookings'
         details = e2_client.post(details_route, [booking_id])
 
         image_route = "/elite2api/api/images/#{details.first['facialImageId']}/data"
```

Comment:
This might fail as well if 'facialImageId' is not in the `details` hash, so might be worth checking that too